skimage/morphology/tests/test_extrema.py

```diff
@@ -217,14 +217,24 @@ def test_empty(self):
         assert result.dtype == np.uint8
 
         result = extrema.local_maxima(np.array([]), indices=True)
-        assert result.size == 0
-        assert result.dtype == np.intp
+        assert isinstance(result, tuple)
+        assert len(result) == 1
+        assert result[0].size == 0
+        assert result[0].dtype == np.intp
+
+        result = extrema.local_maxima(np.array([[]]), indices=True)
+        assert isinstance(result, tuple)
+        assert len(result) == 2
+        assert result[0].size == 0
+        assert result[0].dtype == np.intp
+        assert result[1].size == 0
+        assert result[1].dtype == np.intp
 
     def test_dtypes(self):
         """Test results with default configuration for all supported dtypes."""
         for dtype in self.supported_dtypes:
             result = extrema.local_maxima(self.image.astype(dtype))
-            assert_equal(result, self.expected_default)
+            assert_array_equal(result, self.expected_default)
```

Comment:
Why did you prefer ``assert_array_equal`` here? ``assert_equal`` calls ``assert_array_equal`` when the arguments are arrays.

Reply:
Just a hunch. I felt that it is more explicit, stricter and more inline with what we are actually testing. I have the impression that's advantageous for unit tests. Should I revert this?

Reply:
I don't mind either convention, but I would in general discourage you and other contributors from making changes like this one, which touch a lot of lines for just a matter of taste, as it creates noise in the git history for no substantial benefit. This seems small but, since I've spent a few times chasing down decisions by looking at git histories, I can attest that changes like this *are* disruptive to this kind of "archeological" work. =)

Reply:
Your arguments against this change are way better than my hunch! I'll revert it. :D